static/js/pages/committee-single.js

```diff
@@ -180,6 +180,12 @@ var disbursementRecipientIDColumns = [
   }
 ];
 
+var expendituresColumns = [
+  tables.currencyColumn({data: 'total'}),
+  columns.supportOpposeColumn,
+  tables.committeeColumn({data: 'candidate', className: 'all'}),
```

Comment:
This needs to be ` tables.candidateColumn({data: 'candidate', className: 'all'}),`